airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_source.py

```diff
@@ -113,19 +113,21 @@ def test_get_custom_insights_streams(self, api, config):
 
     def test_get_custom_insights_action_breakdowns_allow_empty(self, api, config):
         config["custom_insights"] = [
-            {"name": "test", "fields": ["account_id"], "breakdowns": ["ad_format_asset"], "action_breakdowns": []},
+            {"name": "test", "fields": ["account_id"], "breakdowns": ["ad_format_asset"], "action_breakdowns": [], "action_report_time": "mixed"},
```

Comment:
Why haven't been tested with old config for custom_insights? 
`{"name": "test", "fields": ["account_id"], "breakdowns": ["ad_format_asset"], "action_breakdowns": []}`

Reply:
I thought that adding a new parameter with a default value, the output would be the same, but if it isn't like that, I'll remove this parameter on unit tests where I added it

Reply:
I did some changes to keep old config for some unit tests which I modified